docs/SetupGuide_Python.md

```diff
@@ -115,23 +113,27 @@ Note: This tutorial requires that a SQL database is setup as shown in [Create a
 
 #### Query String
 
-_TODO_
+The input binding executes the `SELECT * FROM Products WHERE Cost = @Cost` query. The Parameters argument passes the `{cost}` specified in the URL that triggers the function, `getproducts/{cost}`, as the value of the `@Cost` parameter in the query. CommandType is set to `Text`, since the constructor argument of the binding is a raw query.
+
+https://github.com/Azure/azure-functions-sql-extension/tree/main/samples/samples-python/GetProducts
 
 #### Empty Parameter Value
 
-_TODO_
+In this case, the parameter value of the `@Name` parameter is an empty string.
+
+https://github.com/Azure/azure-functions-sql-extension/tree/main/samples/samples-python/GetProductsNameEmpty
 
 #### Null Parameter Value
 
+If the `{name}` specified in the `getproducts-namenull/{name}` URL is "null", the query returns all rows for which the Name column is `NULL`. Otherwise, it returns all rows for which the value of the Name column matches the string passed in `{name}`
+
 _TODO_
 
 #### Stored Procedure
 
-_TODO_
+`SelectProductsCost` is the name of a procedure stored in the user's database. In this case, CommandType is `StoredProcedure`. The parameter value of the `@Cost` parameter in the procedure is once again the `{cost}` specified in the `getproducts-storedprocedure/{cost}` URL.
 
-#### IAsyncEnumerable
-
-_TODO_
+https://github.com/Azure/azure-functions-sql-extension/tree/main/samples/samples-python/GetProductsStoredProcedure
```

Comment:
nit: One thing I just learned about is that for "proper" markdown raw links like this should be put in <>
```suggestion
<https://github.com/Azure/azure-functions-sql-extension/tree/main/samples/samples-python/GetProductsStoredProcedure>
```